src/Unleash/Utilities/ToggleBootstrapFileProvider.cs

```diff
@@ -15,7 +16,14 @@ internal ToggleBootstrapFileProvider(string filePath, UnleashSettings settings)
 
         public string Read()
         {
-            return settings.FileSystem.ReadAllText(filePath);
+            try
+            {
+                return settings.FileSystem.ReadAllText(filePath);
+            }
+            catch (FileNotFoundException)
+            {
+                return string.Empty;
+            }
```

Comment:
Maybe this used to use the old mock thing under the hood that does things weirdly, and the test verified the incorrect mock 🤷

Reply:
I thought about that but this seemed to call out to the real fs. If there's a swap out for the mock fs I can't spot it